addons/web/static/src/search/search_panel/search_view.scss

```diff
@@ -148,62 +144,49 @@
     }
 }
 
+.o_add_buttons {
```

Comment:
Always scope generic selectors. 
`o_add_buttons {}` should be `. o_searchview  .o_add_buttons {}` or smthg ;)